wp-includes/rest-api/auth/class-wp-rest-key-pair.php

```diff
@@ -80,7 +80,7 @@ public static function get_rest_uri() {
 			$prefix = rest_get_url_prefix();
 		}
 
-		return sprintf( '/%s/%s/%s', $prefix, self::_NAMESPACE_, self::_REST_BASE_ );
+		return sprintf( '%s/%s/%s/%s', get_home_url(), $prefix, self::_NAMESPACE_, self::_REST_BASE_ );
```

Comment:
Why add the home url here? This will break the tests and would require some additional modifications.